src/routes/AuthorityViewRoute/AuthorityViewRoute.test.js

```diff
@@ -0,0 +1,27 @@
+import { MemoryRouter } from 'react-router-dom';
+
+import { render } from '@testing-library/react';
+
+import AuthorityViewRoute from './AuthorityViewRoute';
+import Harness from '../../../test/jest/helpers/harness';
+
+jest.mock('../../views/AuthorityView/AuthorityView', () => () => <div>AuthorityView</div>);
+
+const renderAuthorityViewRoute = () => render(
+  <MemoryRouter>
+    <Harness>
+      <AuthorityViewRoute />
+    </Harness>
+  </MemoryRouter>,
+);
+
+describe('Given AuthorityViewRoute', () => {
+  beforeEach(() => {
+  });
```

Comment:
Do we need this empty beforeEach here?

Reply:
Nope, I'll remove it, thanks for noticing